wiringPiD/Makefile

```diff
@@ -41,13 +41,13 @@ LIBS    = -lwiringPi -lwiringPiDev -lpthread -lrt -lm -lcrypt
 # May not need to  alter anything below this line
 ###############################################################################
 
-SRC	=	wiringpid.c network.c runRemote.c daemonise.c
+SRC	=	wiringpi-opid.c network.c runRemote.c daemonise.c
 
 OBJ	=	$(SRC:.c=.o)
 
-all:		wiringpid
+all:		wiringpi-opid
```

Comment:
wiringpid-opi seems better

Reply:
actually this one was a mistake in the global rename - I did undo this one. the wiringpid is not built